numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -32,6 +32,9 @@ def get(name):
             For other keyword-only arguments, see the
             :ref:`ufunc docs <ufuncs.kwargs>`.
     """).strip(),
+    'BROADCASTABLE': ("If ``x1.shape != x2.shape``, they must be broadcastable"
+                      " to a common shape (which may be the shape of one or "
+                      "the other)."),
```

Comment:
Broadcasting does not necessarily result in a shape matching one or the other, e.g., broadcasting arrays with shape `(5, 1)` and `(1, 6)` gives you arrays with shape `(5, 6)`.

I think it would be better to drop the parenthetical.

Reply:
Indeed... thanks!